.eslintignore

```diff
@@ -0,0 +1 @@
+/dist
```

Comment:
Why do you use ignore for the dist folder if in your tsconfig "outDir": "build"?

Reply:
This file was taken from the previous project and wasn't properly revised . Now `/dist` has been replaced by `/build`.